index.html

```diff
@@ -47,5 +53,234 @@ <h2>LET’S CONNECT</h2>
+              </li>
+              <li>
+                <a href="#"> <img src="icons/Linkedin icon.svg" /> </a>
+              </li>
+              <li>
+                <a href="#"> <img src="icons/m.svg" /> </a>
+              </li>
+              <li>
+                <a href="#"> <img src="icons/github.svg" /> </a>
+              </li>
+              <li>
+                <a href="#"> <img src="icons/peace.svg" /> </a>
+              </li>
+            </ul>
+          </div>
+          <div class="actions">
+            <button class="action action-text btn">Get my resume</button>
+          </div>
+        </div>
+        <div class="skill-list">
+          <div class="languages">
+            <div class="language-title title-arrow">
+              <span> Languages </span>
+              <span> <img src="icons/down-arrow.png" alt="" /> </span>
+            </div>
+            <hr class="separator" />
+            <div class="language-wrapper">
+              <ul class="language-list">
+                <li class="language">
+                  <span class="langauge-logo">
+                    <img src="assets/images/js.png" alt="js" />
+                  </span>
+                  <span class="language-name"> Javascript </span>
+                </li>
+                <li class="language">
+                  <span>
+                    <img src="assets/images/html elipse.png" alt="html" />
+                  </span>
+                  <span class="language-name"> HTML </span>
+                </li>
+                <li class="language">
+                  <span>
+                    <img src="assets/images/css elipse.png" alt="css" />
+                  </span>
+                  <span class="language-name"> CSS </span>
+                </li>
+              </ul>
+            </div>
+          </div>
+          <!-- Placeholder -->
+          <div class="frameworks">
+            <div class="framework-title title-arrow">
+              <span> Framework </span>
+              <span> <img src="icons/left-arrow.svg" alt="" /> </span>
+            </div>
+            <hr class="separator" />
+            <div class="framework-wrapper">
+              <ul class="framework-list title-arrow">
+                <li class="framework">
+                  <span class="framework-logo"></span>
+                  <span class="framework-name"> </span>
+                </li>
+              </ul>
+            </div>
+          </div>
+          <div class="skill">
+            <div class="skill-title">
+              <span>Skills</span>
+              <span><img src="icons/left-arrow.svg" alt="" /></span>
+            </div>
+            <hr class="separator" />
+            <div class="skill-wrapper">
+              <ul class="skill-list">
+                <li class="skill">
+                  <span class="skill-logo"></span>
+                  <span class="skill-name"></span>
+                </li>
+              </ul>
+            </div>
+          </div>
```

Comment:
- [ ] to be semantically correct, the skills list (eg : Languages, Framework, etc ..) should be grouped by `<ul>` list and `<li>` per each skill , kindly fix it 🙏

Reply:
Hi @ichala, I tried to correct it can you check it, please?  
- If you have material on semantics please share it with me.